### DISCUSSION TITLE
Filters not working on custom views for list operation (Pro)

### DISCUSSION THREAD

**Discussion by ahmadmobaraki** [Laravel-Backpack/community-forum#1201] [General] [answered]
I added this custom list view to the end of setupListOperation()

```
 $this->runCustomViews([
            'setupMyCustomView' => __('Technical Information View')
   ]);
```
Then added/removed some columns inside the custom view `setupMyCustomView()`. 

Now how to add filters for this custom view? I want the new filters to be different from the default view.  Tried to remove all filters and added new ones inside the setupMyCustomView(), didn't work. Also tried to add/remove filters after calling ` $this->runCustomViews()...` it didt work either. 

Seems backpack deliberately removes all the filters in custom list views, see the screenshot
 
![image](https://github.com/user-attachments/assets/7f82556b-3aac-4512-a714-5e7a3d11b2a1)

**Comment by karandatwani92** [comment, marked as answer]
Hey @ahmadmobaraki 

Sorry to tell you this, but the custom views do not support filters in the current version.
To meet your needs, you can create multiple CRUDs extending the parent CRUD and use them for the same model.
